frontend_v2/src/app/components/profile/profile.component.html

```diff
@@ -116,6 +116,7 @@
                 <div class="form row">
                   <div class="col-lg-10 col-md-10 col-sm-10 col-xs-10">
                     <app-input
+                      id="auth_token_input"
```

Comment:
@ShauryaAg it would be good to make this a css class

Reply:
Changing this to a class would not make the CSS work for some reason.